jobs/aws/eks-distro-build-tooling/athens-postsubmits.yaml

```
@@ -0,0 +1,64 @@
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+postsubmits:
+  aws/eks-distro-build-tooling:
+  - name: athens-postsubmit
+    always_run: false
+    run_if_changed: "athens/*"
```

Can you do the same for the presubmit?
```suggestion
    run_if_changed: "projects/gomods/athens/*"
```